.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -75,6 +75,7 @@ jobs:
       working-directory: artifacts
       run: |
         Set-PSDebug -Trace 1
+        pip install azure-ai-ml
```

Comment:
why install this? this is not expect to have v2 dependency

Reply:
I added a test to avoid module_name is written incorrectly, where "promptflow.azure" depends on "azure-ai-ml".
![image](https://github.com/microsoft/promptflow/assets/23182548/dcc4a261-e682-4299-9e07-694943685ea1)
![image](https://github.com/microsoft/promptflow/assets/23182548/9c428307-2a2e-445b-8cd5-0153ce5dfc54)

Reply:
done.